### ISSUE TITLE
Bi-directional last.fm favorites sync

### ISSUE THREAD

**Issue by melvyn2** [Taiko2k/TauonMusicBox#994]
Adding and removing favorites to last.fm from tauon works great. However, adding or removing favorites from the last.fm website does not update the favorites on tauon.

**Comment by Taiko2k** [comment]
Hmm, a few issues:
  - Last.fm doesn't tell us when a track is unloved. 
  - To know the current state of a track, we need to poll, but I really prefer keep API use to a minimum, I don't like the idea of polling every time a track is played for example.
  - Tauons loves are kind of its own feature, so we're trying to do a 3 way sync which doesn't really work.

Currently in Tauon you can do a one off full sync to you last.fm loves by going MENU > Settings > Accounts > Last.fm: click "Clear" next to "Get User Loves", then click "Get User Loves". But be warned this is a very slow process.

Also I want to avoid further integration with last.fm. 

So I don't really have a solution to this except to suggest avoid loving/unloving tracks on the website.

**Comment by melvyn2** [comment]
I understand the avoiding integration aspect- if a PR were made for this, would having it be an optional feature for last.fm work?
The reason I would like to handle last.fm side changes is because another music client that I use integrates with their favorites, so it can act as an intermediary between the two and I get full favorites sync. They achieve it by user-configurable polling, though I think checking on scrobble (or maybe file/tag rescan) seems like a better solution.

**Comment by Taiko2k** [comment]
OK then, I've now committed this change, adding this setting:

![Screenshot from 2023-06-05 00-21-30](https://github.com/Taiko2k/TauonMusicBox/assets/17271572/42956bd8-5dff-4ea9-9be6-4b87fd92c522)

So, with the rescan, it'll do it with the rescan function via the track context menu, but not the playlist tab context menu or import. 

Enabling this setting does slow down the tag rescan somewhat.

Also, I noticed a bug in my testing, possibly when last.fm corrects artist name or track name, it doesn't get the correct loved status. Not sure who's fault this is. Out of precaution, this new function will skip over corrected tracks when a name correction is detected.
